jetty-ee9/jetty-ee9-servlet/src/test/java/org/eclipse/jetty/ee9/servlet/DefaultServletTest.java

```diff
@@ -138,18 +134,29 @@ public void init(WorkDir workDir) throws Exception
     @AfterEach
     public void destroy() throws Exception
     {
-        server.stop();
-        server.join();
+        if (server != null)
+        {
+            server.stop();
+            server.join();
+        }
         assertThat(FileSystemPool.INSTANCE.mounts(), empty());
     }
 
     @Test
     public void testListingWithSession() throws Exception
     {
-        ServletHolder defholder = context.addServlet(DefaultServlet.class, "/*");
-        defholder.setInitParameter("dirAllowed", "true");
-        defholder.setInitParameter("redirectWelcome", "false");
-        defholder.setInitParameter("gzip", "false");
+        Path docRoot = workDir.getEmptyPathDir().resolve("docroot");
```

Comment:
These 2 lines seem to be repeated by every test, move into `@BeforeEach`?

Reply:
No, as this logic needs is not consistent across the test cases, only about 70% of them, and the new testcases coming from Issue #11925 will not use this as well.